docs/docs/dev_docs/aztecjs/main.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Aztec.js
+---
+
+If you are looking for the API reference, go [here](../../apis/aztec-js/index.md).
+
+## Introduction
+
+Aztec.js is a library that provides APIs for managing accounts and interacting with contracts on the Aztec network. It communicates with the [Private eXecution Environment (PXE)](https://docs.aztec.network/apis/pxe/interfaces/PXE) through a `PXE` implementation, allowing developers to easily register new accounts, deploy contracts, view functions, and send transactions.
+
+## Usage
+
+### Create a new account
+
+```typescript
+import { getSchnorrAccount } from "@aztec/aztec.js";
```

Comment:
These code snippets should reference source code using #include_code so we know if they get out of date.

Reply:
these arent in the codebase anywhere